### PR TITLE
Add support for vim-lsc (fixes #170)

### LINKED ISSUE #170: Support vim-lsc
Any plans to support [vim-lsc](https://github.com/natebosch/vim-lsc/blob/master/README.md)?

Thanks

### PR DESCRIPTION
**Pull request by pwntester**

### CODE REVIEW

File: autoload/vista/executive/vim_lsc.vim [hunk 1]
```diff
@@ -0,0 +1,94 @@
+" Copyright (c) 2019 Liu-Cheng Xu
```
Comment (liuchengxu) [inline autoload/vista/executive/vim_lsc.vim]: Could change to your name :).

**Comment by liuchengxu** [comment]
If this works for you, I'm ok then.

File: autoload/vista/executive/vim_lsc.vim (hunk 1 above)
Comment (pwntester) [reply autoload/vista/executive/vim_lsc.vim]: Its ok, its mostly your code anyhow :)

**Comment by pwntester** [comment]
> If this works for you, I'm ok then.

Yep, works great

**Comment by liuchengxu** [comment]
Merged, thanks!

**Pull request merged**